mscore/seq.cpp

```diff
@@ -1314,19 +1314,23 @@ void Seq::stopNotes(int channel, bool realTime)
                   putEvent(event);
             else
                   sendEvent(event);
-      };
+            };
+      auto sendAllNotesOff = [send](int channel) {
+            for (int note = 0; note < 128; note++)
```

Comment:
Why is it 128? Afaiu, this argument in `NPlayEvent` constructor is a pitch value. Why is the range of that values `[0, 128)`?

Reply:
I believe pitches need to be between 0 and 127 (inclusively)?

Reply:
Programs - yes. Pitches - not at all. I'm going deeper to the semantic of that parameters. Will make a rebased version of this PR once I clearly understand how it works.

Reply:
The midi signal for turning a note off uses an unsigned, seven bit parameter for pitch, which the largest number that can be represented is `2^7 - 1`, or 127.

To make sure, I downloaded the 1996 specification, and it has notes being represented as number from 0 to 127, with C4 on a piano being at note 60. While this is what the 1996 specification says, it's possible the appenda changed some things.

Reply:
That's what I meant actually;-)